spec/controllers/user_spec.rb

```diff
@@ -19,11 +17,14 @@
         post :create, params: user
         expect(User.count).to eq(count + 1)
       end
-      it 'Redirects to home page' do
+      it 'Redirects to new page' do
         post :create, params: user
-
         expect(response.status).to eq(302)
```

Comment:
This test checks that there is a HTTP 302 response, it doesn't check which page is redirected to though. You may want to add this test case in, as well as the unhappy path - eg. that it doesn't redirect.

Reply:
Changed test case, and added unhappy path